src/sentry/receivers/features.py

```diff
@@ -146,11 +145,20 @@ def record_issue_assigned(project, group, user, **kwargs):
     )
 
 
-@issue_resolved_in_release.connect(weak=False)
-def record_issue_resolved_in_release(project, group, user, resolution_type, **kwargs):
-    FeatureAdoption.objects.record(
-        organization_id=project.organization_id, feature_slug="resolved_in_release", complete=True
-    )
+@issue_resolved.connect(weak=False)
+def record_issue_resolved(organization_id, project, group, user, resolution_type, **kwargs):
+    if resolution_type in ('in_next_release', 'in_release'):
+        data_resolution_type = 'resolved_in_release'
+        FeatureAdoption.objects.record(
+            organization_id=organization_id, feature_slug="resolved_in_release", complete=True
+        )
+    elif resolution_type == 'in_commit':
+        data_resolution_type = 'with_commit'
```

Comment:
Is there a reason the analytics `resolution_type` has to differ from what we're using elsewhere (ie. `in_commit`)?

Reply:
It's def only for [data consistency](https://redash.getsentry.net/queries/1537#2127) on how we categorize now. I don't know why it was chosen originally/not sure if there really was a reason to deviate but we've only been capturing this since September so we _could_ switch it up now